src/com/library/backlog/User.java

```diff
@@ -34,68 +27,23 @@ public String getPassword(){
 		return password;
 	}
 	
-	public static UserRole getRole() {
-
+	public static UserRole getRole(){
 		return role;
 	}
 	
+
+	
 	//adding user
-	public void addNewUser(String username, String password){
+	public void addNewUser(ArrayList<User> userList){
+		
 		User newUser = new User(UserRole.USER, username, password);
-		userList = new ArrayList<User>();
+		
 		if(userList.contains(newUser)){
 			System.out.println("The user already exist, please enter another");
 		}
 		else{
 			userList.add(newUser);
+			}
 		}
-	}
 	
-	public void login(){
-		
```

Comment:
remove this method from here and pit it in library.java class
Why you have two login methods in to files ?